books/free-programming-books-hi.md

```diff
@@ -8,3 +8,8 @@
 * [C++ Brief Notes \| Hindi](https://ehindistudy.com/2020/12/01/cpp-notes-in-hindi/) - Yugal Joshi
 * [C++ in Hindi \| Hindi](https://www.bccfalna.com/IOC-AllEBooks/CPPinHindi.pdf) - Kuldeep Chand (PDF)
 * [C++ Introduction Book \| Hindi](https://ncsmindia.com/wp-content/uploads/2012/04/c++-hindi.pdf) - NCMS India (PDF)
+
+
+### <a id="javascript"></a>Js
+
+* [Ultimate JavaScript cource \| Hindi](https://youtube.com/playlist?list=PLu0W_9lII9ahR1blWXxgSlL4y9iQBnLpR) - Code With Harry
```

Comment:
This should be placed under free-programming-books/courses/free-courses-hi.md
and the same course can be found on LINE 276 (in the javascript part) of this free-courses-hi.md file.
It cause duplicate.